run.go

```diff
@@ -114,11 +145,28 @@ func doRun(ctx *cli.Context) error {
 			return errors.Wrapf(err, "Failed creating VM instance")
 		}
 
+		log.Debugf("Adding nics...")
+		for nidx := range vm.Nics {
+			vmNic := vm.Nics[nidx]
+			log.Debugf("Adding nic %s network %s", vmNic.ID, vmNic.Network)
+			var nicNetwork NetworkDef
+			for netidx := range suite.Networks {
+				network := suite.Networks[netidx]
+				if network.Name == vmNic.Network {
+					nicNetwork = network
+					break
+				}
+			}
```

Comment:
Can we assert here that nicNetwork was in fact found?

Reply:
Yes, I'll check it